nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -1176,6 +1176,146 @@ Secure Prompt
 
 If you prefer not to provide the password or raw key in the command-line invocation of the tool, leaving these arguments absent will prompt a secure console read of the password (by default) or raw key (if the `-r` flag is provided at invocation).
 
+[[admin-toolkit]]
+Administrative Tools
+--------------------
+The admin toolkit contains command line utilities for administrators to support NiFi maintenance in standalone
+and clustered environments. These utilities include:
+
+* Notify -- The notification tool allows administrators to send bulletins to the NiFi UI using the command line.
+* Node Manager -- The node manager tool allows administrators to perform a status check on a node as well as to connect, disconnect, or remove nodes that are part of a cluster.
+
+The admin toolkit is bundled with the nifi-toolkit and can be executed with scripts found in the _bin_ folder.
+
+=== Prerequisites for Running Admin Toolkit in a Secure Environment
+For secured nodes and clusters, two policies should be configured in advance:
+
+* Access the controller – A user that will have access to these utilities should be authorized in NiFi by creating an “access
+the controller” policy ('/controller') with both view and modify rights.
```

Comment:
`/controller` doesn't appear to use the tick marks per @alopresto comment "Should mark /controller with '``'.